components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/TabViewHolder.kt

```diff
@@ -45,7 +46,8 @@ class TabViewHolder(
             tab.url
         }
 
-        tabView.text = title
+        titleView.text = title
```

Comment:
We may end up showing the URL twice in the title and URL portions of the tab view when we don't have a title, but let's get to that in a follow-up when we talk to UX about it.